lib/msal-angular/README.md

```diff
@@ -60,150 +136,161 @@ export function loggerCallback(logLevel, message, piiEnabled) {
                   piiLoggingEnabled: true,
                 })]
            })
-           
-  export class AppModule { }
-````
-## Config options
 
-<b>clientID(Mandatory)</b>: The clientID of your application, you should get this from the application registration portal.
+  export class AppModule { }
+```
 
-<b>redirectUri(Optional)</b> : The redirect URI of your app, where authentication responses can be sent and received by your app. It must exactly match one of the redirect URIs you registered in the portal, except that it must be URL encoded.
+* **redirectUri** : The redirect URI of your app, where authentication responses can be sent and received by your app. It must exactly match one of the redirect URIs you registered in the portal, except that it must be URL encoded.
 Defaults to `window.location.href`.
- 
-<b>authority(Optional)</b>:  A URL indicating a directory that MSAL can use to obtain tokens.
-   * - In Azure AD, it is of the form https://&lt;instance>/&lt;tenant&gt;, where &lt;instance&gt; is the directory host (e.g. https://login.microsoftonline.com) and &lt;tenant&gt; is a identifier within the directory itself (e.g. a domain associated to the tenant, such as contoso.onmicrosoft.com, or the GUID representing the TenantID property of the directory)
-   * - In Azure B2C, it is of the form https://&lt;instance&gt;/tfp/&lt;tenantId&gt;/&lt;policyName&gt;/
-   * - Default value is: "https://login.microsoftonline.com/common"
-   
-<b>validateAuthority(Optional)</b> :Validate the issuer of tokens. Default is true.
 
-<b>cacheLocation(Optional)</b>: Sets browser storage to either 'localStorage' or sessionStorage'. Defaults is 'sessionStorage'.
+* **authority** :  A URL indicating a directory that MSAL can use to obtain tokens.
+        * In Azure AD, it is of the form https://<instance>/<tenant>, where <instance> is the directory host (e.g. https://login.microsoftonline.com) and <tenant> is an identifier within the directory itself (e.g. a domain associated to the tenant, such as contoso.onmicrosoft.com, or the GUID representing the TenantID property of the directory)
```

Comment:
all the markup style angular brackets need to be html encoded, like &lt; and &gt;